### PR TITLE
Rule for $Values

### PR DESCRIPTION
**Pull request by mrm007**
[Docs for `$Keys`][keys]
[Docs for `$Values`][values]

[keys]: https://flow.org/en/docs/types/utilities/#toc-keys
[values]: https://flow.org/en/docs/types/utilities/#toc-values

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -14,7 +14,14 @@ module.exports = {
     'no-console': 0,
     'no-plusplus': 0,
     'no-prototype-builtins': 0,
-    'no-restricted-syntax': [2, 'WithStatement'],
+    'no-restricted-syntax': [
+      2,
+      'WithStatement',
+      {
+        selector: 'GenericTypeAnnotation[id.name=$Values]',
+        message: `$Values doesn't do what you think it does. RTFM and use $Keys instead – https://flow.org/en/docs/types/utilities/`
```
Comment (loicplaire) [inline index.js]: I have no idea what RTFM so I would say it probably shouldn't be in the message. (some people might also find this offensive after googling it 😛)

Comment (mrm007) [reply index.js]: I means "read the friendly manual" 🙂. I personally find shit code (that can be avoided by reading the manual) to be more offensive than seeing *fucking* after a Google search, but that's just me.

Comment (mrm007) [reply index.js]: What message would you suggest?

Comment (loicplaire) [reply index.js]: I think some people would find this offensive, we don't want to shame people for making mistakes. I know you, so it doesn't offend me and having some personality in the messages is good but I think we should be careful as the team grows!

Comment (loicplaire) [reply index.js]: I would just drop the "RFTM and" and have "Use $Keys instead – https://flow.org/en/docs/types/utilities/"

**Pull request merged**